<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DayRequestWindow</class>
 <widget class="QDialog" name="DayRequestWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>362</width>
    <height>307</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Solicitar Días</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../images/Resource_ha.qrc">
    <normaloff>:/F1/fromHelyx/eye16.png</normaloff>:/F1/fromHelyx/eye16.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="2">
    <widget class="QCheckBox" name="diaUnico">
     <property name="text">
      <string>Sólo un día</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="2">
    <widget class="QDateEdit" name="fechaDesde">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Fecha Hasta:</string>
     </property>
    </widget>
   </item>
   <item row="7" column="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Fecha Desde: </string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Razón:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QDateEdit" name="fechaHasta">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="6" column="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="2">
    <widget class="QPlainTextEdit" name="razon">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>155</width>
       <height>90</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QComboBox" name="diaUnicoComboBox">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <item>
      <property name="text">
       <string>Todo el día</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Medio día (mañana)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Medio día (tarde)</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Tipo:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QComboBox" name="tipo">
     <item>
      <property name="text">
       <string>Licencia</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Ausencia</string>
      </property>
     </item>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../images/Resource_ha.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DayRequestWindow</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>341</x>
     <y>206</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>129</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DayRequestWindow</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>341</x>
     <y>212</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>129</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>diaUnico</sender>
   <signal>stateChanged(int)</signal>
   <receiver>DayRequestWindow</receiver>
   <slot>changeUniqueDay()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>263</x>
     <y>69</y>
    </hint>
    <hint type="destinationlabel">
     <x>175</x>
     <y>64</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>razon</sender>
   <signal>textChanged()</signal>
   <receiver>DayRequestWindow</receiver>
   <slot>controlLength()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>263</x>
     <y>126</y>
    </hint>
    <hint type="destinationlabel">
     <x>175</x>
     <y>111</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>changeUniqueDay()</slot>
  <slot>controlLength()</slot>
 </slots>
</ui>
